plugin/web/extensions/location/index.tsx

```diff
@@ -0,0 +1,80 @@
+// import { Typography } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+import React from "react";
```

Comment:
Unneeded

Reply:
done

Reply:
ca838bb63dff7bc9efdcc1e7ad0376fefc3c3404

Reply:
This hasn't been removed yet. 
Because of Vite's `react` plugin we don't need to write our React import. It'll do it for us on build!

Reply:
Still not resolved

Reply:
i though you mean the hinted import , done a514034c35204be4d7867358f7ba49a083167418